src/modules/services/submodule/attestation/views/ViewServicesAttestationCreate.vue

```diff
@@ -166,20 +290,57 @@ export default {
       slideDirection.value = 'slide-from-left';
       isNext.value = false;
     };
-
+    const onFinish = () =>{
+      currentPage.value = 1;
+      naviToProcessing();
+      clearInput();
+      checkInProgress.value = false
+    }
     const clickNext = () => {
       slideDirection.value = 'slide-from-right';
       isNext.value = !isNext.value; attestationTrigger.value = true;
     };
 
+    watch(isUploaded, (n, o) => {
+      if (n === false){
+         attestationTrigger.value = false
+      }else{
+        attestationTrigger.value = true
+      }
+    })
+    watch(confirmedTxLength, (n, o) => {
+      if (n-o == 1){
+        proceedCertificate(apostilleTemp.value)
+      }
+    })
```

Comment:
this logic is actually wrong, you will need to check on the confirmed transaction hash list.